src/renderer/components/settings/TrayColorPicker.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-License-Identifier: GPL-3.0
+ * Vesktop, a desktop app aiming to give you a snappier Discord Experience
+ * Copyright (c) 2023 Vendicated and Vencord contributors
+ */
+
+import "./traySetting.css";
+
+import { Margins } from "@vencord/types/utils";
+import { findByCodeLazy } from "@vencord/types/webpack";
+import { Forms } from "@vencord/types/webpack/common";
+import { setCurrentState } from "renderer/patches/tray";
+
+import { SettingsComponent } from "./Settings";
+
+const ColorPicker = findByCodeLazy(".Messages.USER_SETTINGS_PROFILE_COLOR_SELECT_COLOR", ".BACKGROUND_PRIMARY)");
+
+const presets = [
+    "#3DB77F", // discord default ~
+    "#F6BFAC" // Vesktop inpired
+];
+
+export const trayIconPicker: SettingsComponent = ({ settings }) => {
+    if (!settings.tray) return null; // how to disable instead of hiding?
```

Comment:
is this actually necessary to change? returning null is the correct way in react to render nothing

Reply:
But in settings we also have "Minimize to tray" which disables when Tray is off and I wanted to achieve similar behavior for this too. Because one element just *grays* as inactive and another (mine) disappearing at all